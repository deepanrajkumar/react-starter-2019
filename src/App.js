import React from "react";
import Main from "./Components/Main";
import ErrorBoundary from "./Components/Shared/ErrorBoundary";

const App = props => (
  <ErrorBoundary>
    <Main {...props} />
  </ErrorBoundary>
);

export default App;
