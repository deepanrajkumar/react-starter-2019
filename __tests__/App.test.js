import React from "react";
import renderer from "react-test-renderer";
import App from "../src/App";

test("should render the app", () => {
  const main = renderer.create(<App />).toJSON();
  expect(main).toMatchSnapshot();
});
