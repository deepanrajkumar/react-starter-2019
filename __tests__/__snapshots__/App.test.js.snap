// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the app 1`] = `
<div
  className="new-world"
>
  !! Hello new world !!
</div>
`;
